### PR TITLE
Added simulation properties isDraft and status to search index

### PR DESCRIPTION
**Pull request by crisely09**
Properties to be queried for SynaptomeSimulation and SingleNeuronSimulation types.

### CODE REVIEW

File: tests/docker/config/search/construct-query.sparql [hunk 1]
```diff
@@ -242,6 +242,8 @@ CONSTRUCT {
   ?alias :singleNeuronSimulation  ?singleNeuronSimulation  .
                                   ?singleNeuronSimulation  :injectionLocation                ?injectionLocation .
                                   ?singleNeuronSimulation  :recordingLocation                ?recordingLocation .
+                                  ?singleNeuronSimulation  :isDraft                          ?isDraft .
+                                  ?singleNeuronSimulation  :status                           ?status  .
```
Comment (imsdu) [inline tests/docker/config/search/construct-query.sparql:246]: `status` is already used by simulation campaigns, are we sure they are not clashing each other ?

Comment (crisely09) [reply tests/docker/config/search/construct-query.sparql:246]: are they indexed together? how is the batch constructed?

Comment (crisely09) [reply tests/docker/config/search/construct-query.sparql:246]: is there a way to test this?

Comment (imsdu) [reply tests/docker/config/search/construct-query.sparql:246]: They can as the batch takes the state log in its chronological order

Comment (crisely09) [reply tests/docker/config/search/construct-query.sparql:246]: then isn't it the same for `Simulation` VS `SimulationCampaign`, they both use status

**Review by imsdu — APPROVED** [review]

**Pull request merged**